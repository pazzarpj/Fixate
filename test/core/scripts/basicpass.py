from fixate.core.common import TestClass
from fixate.core.ui import user_ok, user_info
from fixate.core.checks import *

__version__ = "1"


class SimpleTest(TestClass):
    """Simple passing test"""

    def test(self):
        chk_true(True, "It is True!")


TEST_SEQUENCE = [SimpleTest()]