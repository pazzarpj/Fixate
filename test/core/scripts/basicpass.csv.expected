0,Sequence,started=20181009-203855,fixate-version=0.4.0dev,test-script-name=basicpass,report-format=3,index_string=None
0.00,Test 1,start,Simple passing test,
0.00,Test 1,check1,true,It is True!,PASS,True
0.00,Test 1,end,PASS,checks-passed=1,checks-failed=0
0.01,Sequence,ended=20181009-203855,Finished,tests-passed=1,tests-failed=0,tests-error=0,tests-skipped=0,sequence=PASSED